lib/screens/home/history_page_view.dart

```diff
@@ -104,7 +104,7 @@ class _HistoryPageViewState extends State<HistoryPageView> {
                   maxRadius: 28.0,
                   foregroundImage: CachedNetworkImageProvider(employee?.avatarAttachmentUrl ?? ""),
                   child: Text(
-                    getInitials(employeeName, 2),
+                    getInitials(employeeName, 2).toUpperCase(),
```

Comment:
@sriLatha2328 toUpperCase function to be moved inside the getInitials function so that it handles the upper cases wherever used.